apis/python/src/tiledbsoma/types.py

```diff
@@ -31,3 +32,21 @@
 
 SparseCoordinates = Union[int, slice, Tuple[int, ...], List[int], pa.IntegerArray]
 SparseNdCoordinates = Tuple[DenseCoordinates, ...]
+
```

Comment:
We are accepting the burden of keeping these comments up to date (which isn't bad), but if you are adding them, I suggest:
* make them real docstrings so the user benefits from them as well
* add the same thing to the other coordinates, as they all share common principles.

Reply:
@bkmartinjr I should remove them here -- they are duplicate information I left in as crib notes while writing out the union-types and forgot to remove them.